extract-test-code/src/ExtractTestCode.elm

```diff
@@ -205,13 +196,52 @@ extractFromTestFunction : List String -> List Expression -> List ( String, Expre
 extractFromTestFunction descriptions expressions =
     case expressions of
         (Literal name) :: (LambdaExpression test) :: [] ->
-            [ ( (name :: descriptions)
-                    |> List.reverse
-                    |> List.drop 1
-                    |> String.join " > "
-              , Node.value test.expression
-              )
-            ]
+            [ ( buildName name descriptions, Node.value test.expression ) ]
 
         _ ->
             []
+
+
+{-| fuzz functions should have a string parameter to describe
+the test. The full fuzz function is printed because the arguments
```

Comment:
Do we need the full fuzz function for all fuzz tests, or just for fuzzWith?

Reply:
Right now, I have been only using `fuzz` and `fuzzWith`, so we need at least that, but we might as well get them all.

Reply:
I think I'm misunderstanding something again.

For example, this test failure message is perfect, but doesn't include the full function

```
↓ Wordy
✗ should be able to multiply arbitrary integers

Given (0,1)

    Just 1
    ╷
    │ Expect.equal
    ╵
    Just 0
```

Reply:
That's the normal output of `elm-test-rs`, and on the Exercism web editor it will be shown with the full function next to it. 
I would take a screenshot, but it seems the servers are having some kind of trouble at the moment :)